pkg/apis/maistra/v2/telemetry.go

```diff
@@ -2,6 +2,8 @@ package v2
 
 // TelemetryConfig for the mesh
 type TelemetryConfig struct {
+	Enablement `json:",inline"`
```

Comment:
This should be unnecessary.  To disable telemetry, set type to None.

Reply:
See my comment above